1-vault-aws-auth-method.sh

```diff
@@ -84,7 +85,7 @@ pe "rm ~/.vault-token"
 pei ""
 pe "vault auth list"
 pei ""
-pe "vault login token=password"
+pe "vault login token=$VAULT_ROOT_TOKEN"
```

Comment:
Giving the user the ability to specify their root token as an env var. I tried using standard `VAULT_TOKEN` but it somehow kept overriding the token from the `vault login -method=aws`